package-lock.json

```diff
@@ -1,8 +1,14262 @@
 {
   "name": "accessibility-developer-guide",
   "version": "0.0.1",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Now I see the size of the package-lock change, I'm not sure if I'd rather revert this file change.

Reply:
@manuelmeister, revert sounds good to me as there were no changes to the `package.json`.

Reply:
I now know, this is due to npm v7. I'll revert this.

Reply:
@manuelmeister, could you have a look at this as we are currently having a merge conflict?

Franco will do the testing and get back to us.